packages/visualizations/app/styles/navi-visualizations/visualizations/table.less

```diff
@@ -219,3 +224,50 @@
     font-size: @font-size-xsmall;
   }
 }
+
+.table-header-cell__dropdown-container {
+  border: 1px solid @navi-gray-300;
+  box-shadow: 0 3px 8px fade(@navi-black, 38%);
+  font-size: @font-size-base;
+  margin-top: 20px;
+  padding: 10px;
+  max-width: 280px;
+
+  &:before, &:after {
+    border: solid transparent;
+    content: " ";
+    height: 0;
+    position: absolute;
+    pointer-events: none;
+    right: 8px;
+    top: -25px;
+    width: 0;
+  }
+
+  &:before {
+    border-bottom-color: @navi-gray-300;
+    border-width: 13px;
+  }
+
+  &:after {
+    border-bottom-color: @navi-white;
+    border-width: 14px;
+  }
+
+  .number-format-selector__section-header {
+    font-weight: 600;
```

Comment:
We try to use our bold fonts instead of adding font-weight to properties. Could we do that here too pls?
Maybe `font-family-strong` or `font-family-bold`? https://github.com/yahoo/navi/blob/master/packages/core/app/styles/navi-core/variables/fonts.less#L19